javascript/javascript1/week2/index3.js

```diff
@@ -0,0 +1,24 @@
+
+var arrayOfWeekdays = ["Sunday","Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+
+const today=new Date()
+const day=today.getDate()
+
+console.log(today)
+console.log(day)
+
+
+function getEventWeekday(num){
+
+    const result=day+(num%7)   
+
+    if(result<=7)
+      return (arrayOfWeekdays[result])
+     else
+      return (arrayOfWeekdays[(result%7)])  
+}
+
+console.log(getEventWeekday(3))
```

Comment:
Nice usage of the **remainder operator.** 💯 Don't forget that **var** is a little outdated. It's always better to use **const** or **let**.